### PR TITLE
Translation for strings.xml in Spanish

### PR DESCRIPTION
**Pull request by jfacdev**
Add Spanish translation directory and initial file for Spanish translation

### CODE REVIEW

**Comment by coderGtm** [comment]
Great work! So is it done from your side? If there are only backslashes remaining then I will handle it, don't worry!

**Comment by jfacdev** [comment]
It's done on my end!

Juan Araujo

**Comment by coderGtm** [comment]
Awesome

**Comment by coderGtm** [comment]
Just remove the swap file which may have accidentally been included in the commit, before I merge it!

File: app/src/main/res/values-es/.strings.xml.swp
Comment (coderGtm) [inline app/src/main/res/values-es/.strings.xml.swp:1]: unnecessary swap file to be removed!

Comment (jfacdev) [reply app/src/main/res/values-es/.strings.xml.swp:1]: Done!

**Comment by coderGtm** [comment]
_Here comes the Spanish Yantra Launcher..._

**Comment by jfacdev** [comment]
Thanks, I'll check it out!

Juan Araujo

**Comment by jfacdev** [comment]
When do you think it should be available to download the update in Google Play Store for Yantra CLI Launcher Pro?

**Comment by coderGtm** [comment]
Waiting for Hungarian translation for the new strings but I will push the update today anyways. You can expect it by tomorrow!

**Comment by jfacdev** [comment]
Awesome, thanks for the update!

Juan Araujo

**Pull request merged**